apps/review-host/check.sh

```
@@ -0,0 +1,110 @@
+if [ -n "$RUNNING_VERSION" ]; then note "host-service: $RUNNING_VERSION (host $HOST_ID)"; else fail "host-service is not answering"; fi
+
+TAG=$(curl -sf -m 20 -H "Accept: application/vnd.github+json" \
+  https://api.github.com/repos/superset-sh/superset/releases/latest 2>/dev/null \
+  | python3 -c 'import json,sys;print(json.load(sys.stdin).get("tag_name",""))' 2>/dev/null)
+LATEST="${TAG#desktop-v}"
+if [ -n "$LATEST" ] && [ -n "$RUNNING_VERSION" ]; then
+  if [ "$LATEST" = "$RUNNING_VERSION" ]; then
+    note "version: current with $LATEST"
+  else
+    NEWER=$(printf '%s\n%s\n' "$RUNNING_VERSION" "$LATEST" | sort -V | tail -1)
+    [ "$NEWER" = "$RUNNING_VERSION" ] && note "version: $RUNNING_VERSION is ahead of released $LATEST" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)